src/odemis/driver/xt_client.py

```diff
@@ -2374,58 +2374,56 @@ def _moveTo(self, future: futures.Future, pos: Dict[str, float], rel: bool = Fal
                 # is eventually updated that the stage has moved unexpectedly. So, wait until the reported position is
                 # (1) not identical to the starting position and (2) not too far from the target position.
 
-                # The stage is not moving anymore, but we still want to wait until the position has been updated
-                # TODO: base the timeout on the stage movement time estimation (est. *10)
-                timeout = 10  # s
-                expected_end_time = time.time() + timeout  # s
-                check_pos = True
-
-                # TODO: Include rotation and tilt to check stage movement to target position. Test it on a hardware.
-                # The below check to skip the check on stage movement to target position if it includes r ot t
-                # is temporary. It will be removed once the behavior is extended to all the axes and confirmed to work
-                # on a hardware.
-                if 'r' in pos or 't' in pos:
-                    logging.debug("Position requested to move includes t (rx/tilt) or r (rz/rotation). "
-                                  "Reported position accuracy is not checked.")
-                    check_pos = False
-
                 # Get the target position in absolute coordinates
                 if rel:
                     target_pos = {ax: val + pos[ax] for ax, val in orig_pos.items()}
                 else:
                     target_pos = pos
 
+                # The stage is not moving anymore, but we still want to wait until the position has been updated
+                # TODO: base the timeout on the stage movement time estimation (est. *10)
```

Comment:
There is now a helper function to estimate the stage movement time easily in https://github.com/patrickcleeve2/odemis/blob/9011af695e241acfc07c03b36892107a67778acf/src/odemis/util/driver.py#L260 if you want to complete this TODO